projects/packages/search/src/instant-search/class-instant-search.php

```diff
@@ -620,7 +620,11 @@ protected function template_parts_have_search_block() {
 	 * @param {string} $block_content - the content to append the search block.
 	 */
 	public static function inject_search_widget_to_block( $block_content ) {
-		$search_block = '<!-- wp:search {"label":"Jetpack Search","buttonText":"Search"} /-->';
+		$search_block = sprintf(
+			'<!-- wp:search {"label":"%s","buttonText":"%s"} /-->',
```

Comment:
Why did we change from "Jetpack Search" to "Search"? Also, what are your thoughts on omitting the label altogether?

Reply:
I like the idea to remove the label, the purpose of the block is very obvious.